sql/analyzer/resolve_columns.go

```diff
@@ -316,7 +316,7 @@ func qualifyColumns(ctx *sql.Context, a *Analyzer, n sql.Node, scope *Scope, sel
 		newNode, sameNode, err := transform.OneNodeExprsWithNode(n, func(n sql.Node, e sql.Expression) (sql.Expression, transform.TreeIdentity, error) {
 			evalSymbols := symbols
 			if in, ok := n.(*plan.InsertInto); ok && len(in.OnDupExprs) > 0 && onDupUpdateLeftExprs[e] {
-				evalSymbols = onDupUpdateSymbols
+				evalSymbols = onDupUpdateSymbols // todo: this should see right?
```

Comment:
Looks like this was just a debugging note? Or is there something left todo here?

Reply:
oops just a debugging note, will delete :P